src/AzureIntegration/Directory.Build.targets

```diff
@@ -1,6 +1,17 @@
 <Project>
   <PropertyGroup>
     <RuntimeFrameworkVersion Condition=" '$(TargetFramework)' == 'netcoreapp3.0' ">$(MicrosoftNETCoreAppPackageVersion)</RuntimeFrameworkVersion>
-    <NETStandardImplicitPackageVersion Condition=" '$(TargetFramework)' == 'netstandard2.0' ">$(NETStandardLibrary20PackageVersion)</NETStandardImplicitPackageVersion>
   </PropertyGroup>
+
+  <PropertyGroup>
```

Comment:
Seems like this is boilerplate now, is it possible to pop it up a level into something like `src/Directory.Build.targets`?

Reply:
This entire file will completely go away as we continue to make progress on #4246.